ofl/alkalami/METADATA.pb

```diff
@@ -0,0 +1,21 @@
+name: "Alkalami"
+designer: "SIL International"
+license: "OFL"
+category: "SERIF"
+date_added: "2022-06-10"
+fonts {
+  name: "Alkalami"
+  style: "normal"
+  weight: 400
+  filename: "Alkalami-Regular.ttf"
+  post_script_name: "Alkalami-Regular"
+  full_name: "Alkalami"
+  copyright: "Copyright (c) SIL International, 2015-2022"
+}
+subsets: "latin"
```

Comment:
When arabic set is not detected by the tool, `subsets: "arabic"` needs to be added manually (following alphabetical order, so before latin).